examples/chat/chat.py

```diff
@@ -95,11 +98,21 @@ def main():
         type=str,
         help=f"destination multiaddr string, e.g. {example_maddr}",
     )
+    parser.add_argument(
+        "-l",
+        "--localhost",
+        dest="localhost",
+        action="store_true",
+        help="flag indicating if localhost should be used or an external IP",
+    )
     args = parser.parse_args()
 
+    if not args.port:
+        raise RuntimeError("was not able to determine a local port")
```

Comment:
I set `default` for `args.port`. In this case, is this branch possibly be entered?

Reply:
it would only run this branch if `args.port` was "falsey", but it definitely won't be w/ the default non-zero value